packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Include app-specific information in a custom claim
+---
+In machine to machine flows, if you want to include certain app-specific information in a token claim, you can do so by first adding the metadata into the profile section of the service app. You can access any values that are put inside the app profile using `app.profile` written in [Okta Expression Language](/docs/reference/okta-expression-language/).
+
+To include, for example, the app `label` parameter in a token claim:
+
+* Create a service app with the Profile object.
+* Add a custom claim to your Custom Authorization Server.
+
```

Comment:
@danielzeller-okta  Are there any app-specific claims that could be directly accessed?

Reply:
`app.id`, `app.clientId` and `app.profile.*` are the only app-related properties available in EL for custom claims

Reply:
@susanharper-okta We should call this out. Note: app.id and app.clientId can be directly used as a claims.